js_modules/dagit/packages/core/src/asset-graph/AssetNode.mocks.ts

```diff
@@ -102,14 +102,16 @@ export const LiveDataForNodeNeverMaterialized: LiveDataForNode = {
   partitionStats: null,
 };
 
+const februaryDate = new Date('2022-02-15T00:00:00.000Z').valueOf();
+
 export const LiveDataForNodeMaterialized: LiveDataForNode = {
   stepKey: 'asset1',
   unstartedRunIds: [],
   inProgressRunIds: [],
   lastMaterialization: {
     __typename: 'MaterializationEvent',
     runId: 'ABCDEF',
-    timestamp: `${Date.now()}`,
```

Comment:
All of the tests were expecting "Feb". But now that it's march the test started failing 😅

Reply:
Ahh this will be fixed by https://github.com/dagster-io/dagster/pull/12616 in a minute or two! Rebasing should fix it 🙏